lisa/tools/docker.py

```diff
@@ -65,7 +65,14 @@ def start(self, name: str = "docker") -> None:
     def _install(self) -> bool:
         service_name = "docker"
         if isinstance(self.node.os, Debian):
-            self.node.os.install_packages("docker.io")
+            try:
```

Comment:
ubuntu distro also use the same logic, right?

Reply:
Most ubuntu versions will work with "docker.io" package installation